src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/FeatureFlagNames.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS;
+
+/// <summary>
+///     Provides feature flag names used throughout this project.
+/// </summary>
+internal static class FeatureFlagNames
+{
+    // Use this prefix name for all feature flags where the feature implementations belongs to dotnet project system.
+    public const string Prefix = "ManagedProjectSystem";
```

Comment:
I would avoid pulling this prefix out. It means we cannot search for the full feature flag string to find it's definition in source.

Reply:
Any chance the feature flag names could change to have a delimiter? Like `"ManagedProjectSystem_"`? Then, it is a bit more natural to think to search for a partial name.

Reply:
Fixed in https://github.com/dotnet/project-system/pull/8482